src/lib.rs

```diff
@@ -1149,6 +1235,7 @@ impl Element {
         let Self {
             mut client,
             element,
+            ..
```

Comment:
I don't think this change is needed any more? I'd rather keep it without `..` so that we remember to update it should we add fields in the future.